android/src/main/java/co/nimblehq/kaylabruce/kmmic/android/presentation/screen/surveydetail/SurveyDetailScreen.kt

```diff
@@ -55,9 +58,9 @@ fun SurveyDetailScreen() {
                             Column(
                                 verticalArrangement = Arrangement.SpaceBetween,
                                 modifier = Modifier.fillMaxSize(),
-                                ) {
+                            ) {
                                 Column {
-                                    BackButton()
+                                    BackButton(onBack = { onBack() })
```

Comment:
We could remove the curly bracket here for shorter
```suggestion
                                    BackButton(onBack = onBack)
```